lib/ecto/multi.ex

```diff
@@ -65,7 +65,7 @@ defmodule Ecto.Multi do
           # under keys we used for naming the operations.
         {:error, failed_operation, failed_value, changes_so_far} ->
           # One of the operations failed. We can access the operation's failure
-          # value (like changeset for operations on changesets) to prepare
+          # value (like changeset for operations on changesets) to prepare a
           # proper response. We also get access to results of any operations
           # that succeeded before the indicated operation failed.
```

Comment:
I think it's reasonable to mention it.

Mongo won't be using `Repo.transaction` in the first place, so we don't need to be concerned by that here.